code/game/turfs/turf.dm

```diff
@@ -193,13 +197,17 @@ GLOBAL_LIST_EMPTY(station_turfs)
 		T = GET_TURF_BELOW(src)
 		if(T)
 			T.multiz_turf_del(src, UP)
+
+	biome = null
```

Comment:
what's the purpose of setting it to null here?

Reply:
I just wanted to avoid some hard deletions caused by references not being cleaned up properly, if you tell me that it doesn't matter, I can remove it.

That being said, Lemon brought up some great points, I'm going to remove this new var from turfs anyway. It doesn't have to be a var, at least not with what little usage of it I'm making.